File: Retrofit1_WE/app/src/main/java/com/example/retrofit1_we/ResponseModel.java
package com.example.retrofit1_we;

import androidx.annotation.NonNull;

import com.google.gson.annotations.SerializedName;
import java.io.Serializable;

public class ResponseModel implements Serializable {

	@SerializedName("data")
	private DataModel data;

	@SerializedName("support")
	private SupportModel support;

	public DataModel getData(){
		return data;
	}

	public SupportModel getSupport(){
		return support;
	}

	@NonNull
	@Override
	public String toString() {
		return super.toString();
	}
}

File: Kotlin Projects/GetUserDetailsKotlin/app/src/main/java/com/example/getuserdetails_kotlin/ResponseModel.kt
package com.example.getuserdetails_kotlin

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class ResponseModel(

	@field:SerializedName("data")
	val data: DataModel? = null,

	@field:SerializedName("support")
	val support: SupportModel? = null
)